tests/integration/test_config.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import pytest
+
+from invenio_search import current_search_client as es
+
+
+def _get_suggester_result(response, suggester):
+    return response.json[suggester][0]['options'][0]['_source']
+
+
+def test_literature_suggesters_book_title(api_client, create_record):
+    lit = create_record("lit", data={"titles":[{"title": "Suggested title"}]})
+    es.indices.refresh('records-hep')
+
+    resp = api_client.get('literature/_suggest?book_title=su')
+
+    result_rec_id = _get_suggester_result(resp, 'book_title')['control_number']
+    expected_rec_id = lit['control_number']
+
+    assert result_rec_id == expected_rec_id
+
+
+def test_literature_suggesters_abstract_source(api_client, create_record):
+    lit = create_record("lit", data={"abstracts":[{"value": "Fancy abstract", "source": "WSP"}]})
+    es.indices.refresh('records-hep')
+
+    resp = api_client.get('literature/_suggest?abstract_source=ws')
+
+    result_rec_id = _get_suggester_result(resp, 'abstract_source')['control_number']
```

Comment:
we should check also the other fields defined, some of the are required in the forms